install_requirements.py

```diff
@@ -137,7 +137,7 @@ def python_is_compatible():
     "timm==1.0.7",
     f"torchaudio==2.5.0.{NIGHTLY_VERSION}" if USE_PYTORCH_NIGHTLY else "torchaudio",
     "torchsr==1.0.4",
-    "transformers==4.46.1",
+    "transformers==4.42.4", # TODO update back to 4.46.1 once the error is fixed
```

Comment:
What was the issue with this?

Reply:
Sorry for confusing. This is for our internal CI. Removed.

One more thing: just in case you want to reproduce the performance profiling results right now, we are still checking and working on some related passes. It might be better to wait for our next profiling results to see if the execution times are aligned. Thanks!

Reply:
Sounds good, thanks!

Reply:
Maybe let's bring more CI to oss, so they can be caught when creating the PR